examples/cpp/HelloWorldExample/Monitor.cpp

```diff
@@ -384,6 +384,7 @@ void Monitor::Listener::on_topic_discovery(
         EntityId topic_id,
         const DomainListener::Status& status)
 {
+
     static_cast<void>(domain_id);
```

Comment:
Remove extra line break
```suggestion
{
    static_cast<void>(domain_id);
```